### PR TITLE
update parser

### PR DESCRIPTION
**Pull request by festinalente91**
- update parser
- removed google apikey

### CODE REVIEW

File: src/server/util/books/kakao.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ export default class KakaoBookAPIUtil {
   }
 
   static removeTag(str) {
-    return str.replace(/&lt;b&gt;/g, '');
+    return str.replace(/&lt;b&gt;/g, '').replace(/&lt;\/b&gt;/g, '');
```
Comment (ochanje210) [inline src/server/util/books/kakao.js:179]: so what was the reason for this?

Comment (lyw0149) [reply src/server/util/books/kakao.js:179]: It can remove dirty HTML chars like $lt; or b&gt;.  You saw that yesterday.

**Comment by lyw0149** [comment]
This is good.

**Comment by lyw0149** [comment]
LGTM 

[Approved with PullApprove](https://pullapprove.com/recbook/recbook-server/pull-request/15/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=recbook/recbook-server)

**Pull request merged**